ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PeerSimulator/Tests/Timeouts.hs

```diff
@@ -35,7 +35,7 @@ prop_timeouts = do
   genesisTest <- genChains 0
 
   -- Use higher tick duration to avoid the test taking really long
-  let scSchedule = PointScheduleConfig {pscTickDuration = 1}
+  let scSchedule = PointScheduleConfig {pscTickDuration = 1, psSecurityParam = gtSecurityParam genesisTest}
```

Comment:
or maybe
```suggestion
  let scSchedule = (defaultPointScheduleConfig (gtSecurityParam genesisTest)) {pscTickDuration = 1}
```
or something, to be future proof? (not exactly sure of the name here)

Reply:
sure thing, buddy!